css/apps/photos.scss

```diff
@@ -14,6 +14,15 @@
 				display: none;
 			}
 
+			&__content-right {
+				margin-left: 8px;
+				
+				// disable Share button for Albums
+				form.upload-picker + button.button-vue--icon-only {
+					display: none;
+				}
```

Comment:
empty album will still show 'Share' button since the form.upload-picker doesn't appear there
we could use :has selector solution also so at least in chrome everything works right

Reply:
Done

Reply:
you removed it but i wrote that we could use both solutions at the same time

Reply:
It was not 100% clear from the message, added both ways to hide the share button now.